### PR TITLE
DBT Plugin Enhancements PI07 - Dbt- Vertica 1.8.5 

### PR DESCRIPTION
**Pull request by mkottakota1**
PR Request for merging the changes for dbt-core 1.8.5 & dbt-vertica adapter 1.8.5

### CODE REVIEW

File: dbt/include/vertica/macros/utils/safe_cast.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% macro vertica__safe_cast(field, type) %}
+ {% if type|upper == "GEOMETRY" -%}
+        try_to_geometry({{field}})
```
Comment (ajay-abrol2) [inline dbt/include/vertica/macros/utils/safe_cast.sql:3]: Is this function from Vertica DB?

File: dbt/include/vertica/macros/utils/safe_cast.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    {% elif type|upper == "GEOGRAPHY" -%}
+        try_to_geography({{field}})
```
Comment (ajay-abrol2) [inline dbt/include/vertica/macros/utils/safe_cast.sql:5]: Is this function of Vertica DB?

File: tests/functional/adapter/unit_testing/test_case_insenstivity.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        }
+
+    def test_case_insensitivity(self, project):
+        results = run_dbt(["run"])
+        assert len(results) == 2
+
+        results = run_dbt(["test"])
+
+
+class TestPosgresUnitTestCaseInsensitivity(BaseUnitTestCaseInsensivity):
```
Comment (ajay-abrol2) [inline tests/functional/adapter/unit_testing/test_case_insenstivity.py:48]: rename this testcase as per Vertica

File: tests/functional/adapter/unit_testing/test_types.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            results = run_dbt(["run", "--select", "my_upstream_model"])
+            assert len(results) == 1
+
+            try:
+                run_dbt(["test", "--select", "my_model"])
+            except Exception:
+                raise AssertionError(f"unit test failed when testing model with {sql_value}")
+
+
+class TestPostgresUnitTestingTypes(BaseUnitTestingTypes):
```
Comment (ajay-abrol2) [inline tests/functional/adapter/unit_testing/test_types.py:83]: Rename

**Review by ajay-abrol2 — CHANGES_REQUESTED** [review]
I have reviewed and added few minor comments.

File: dbt/include/vertica/macros/utils/safe_cast.sql (hunk 2 above)
Comment (mkottakota1) [reply dbt/include/vertica/macros/utils/safe_cast.sql]: Removed this function, it's not applicable for Vertica

**Comment by mkottakota1** [comment]
Done the needul

**Pull request closed without merging**